ONSdigital/eq-author-app
# Bug - Stocks - Cannot publish calculated summary

### What is the context of this PR?
Calculate summary is the first page without answers. A lot of the resolvers assumed that all pages had answers and exploded when there was a calculated summary added to the questionnaire.

To resolve this we now have a series of methods in `schema/utils` to access different parts of the questionnaire. This PR aims to switch the resolvers to use them.

### How to review 
1. Create a questionnaire with routing on radio options
2. See that it works
3. Add a calculated summary. See viewing routing and publishing blows up on master. This branch should resolve it.
